packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/App.xaml.cs

```diff
@@ -42,7 +42,9 @@ public App()
             InstanceSettings.Properties.Set(ReactPropertyBagHelper.GetName(null, "Prop1"), 43);
             InstanceSettings.Properties.Set(ReactPropertyBagHelper.GetName(null, "Prop2"), "Hello RNW!");
 
-            PackageProviders.Add(new ReactPackageProvider()); // Includes any modules in this project
+            Microsoft.ReactNative.Managed.AutolinkedNativeModules.RegisterAutolinkedNativeModulePackages(PackageProviders); // Includes any autolinked modules
```

Comment:
is this call ok to make before we modify PackageProviders? should it not happen after we add everything?